backend/src/nodes/nodes/rest/img2img.py

```diff
@@ -0,0 +1,99 @@
+from __future__ import annotations
+
+import numpy as np
+
+from . import category as RESTCategory
+from ...impl.rest import (
+    decode_base64_image,
+    SamplerName,
+    STABLE_DIFFUSION_IMG2IMG_URL,
+    post_async,
+    encode_base64_image,
+)
+from ...node_base import AsyncNodeBase
+from ...node_factory import NodeFactory
+from ...properties.inputs import (
+    TextInput,
+    NumberInput,
+    EnumInput,
+    ImageInput,
+)
+from ...properties.outputs import LargeImageOutput
+
+
+@NodeFactory.register("chainner:rest:sd_img2img")
+class Img2Img(AsyncNodeBase):
+    def __init__(self):
+        super().__init__()
+        self.description = "Modify an image using an external Stable Diffusion service"
+        self.inputs = [
+            ImageInput(),
+            TextInput("Prompt", default="an astronaut riding a horse"),
+            TextInput("Negative Prompt", default=""),
```

Comment:
This default value is invalid. Text inputs enforce a min length of 1 by default. If you want any input to be optional, use its `make_optional()` method. E.g. `TextInput("Negative Prompt").make_optional()`. If the user doesn't assign a value to an optional input, then the `run` of the node will get `None` for this parameter (be sure to denote this in the type hint of the parameter).

Reply:
Ah!  Okay, thank you.